bundles/edu.kit.kastel.sdq.eclipse.grading.view/src/edu/kit/kastel/eclipse/grading/view/commands/AddAnnotationCommandHandler.java

```diff
@@ -81,7 +82,18 @@ public Object execute(ExecutionEvent event) throws ExecutionException {
 
 		// Prevent insertion of a new line because the default keybinding is alt+enter
 		if (event.getTrigger() != null) {
-			((Event) event.getTrigger()).doit = false;
+			// Windows and GTK behave differently in regard to doit
+			if (System.getProperty("os.name").contains("Windows")) {
+				((Event) event.getTrigger()).doit = false;
+			} else {
+				// We are probably under GTK
```

Comment:
Is `SWT.getPlatform() == "gtk"` better to check if the special case should apply?
Under macOS the else case would be triggered as well, where the platform is probably cocoa.

Reply:
Fixed